src/doc/man/cargo-remove.md

```diff
@@ -0,0 +1,94 @@
+# cargo-remove(1)
+{{*set actionverb="Remove"}}
+{{*set nouns="removes"}}
+
+## NAME
+
+cargo-remove - Remove dependencies from a Cargo.toml manifest file
+
+## SYNOPSIS
+
+`cargo remove` [_options_] _dependency_...
+
+## DESCRIPTION
+
+Remove one or more dependencies from a `Cargo.toml` manifest.
+
+After the manifest has been edited, `Cargo.lock` will be updated to reflect any changes.
```

Comment:
I feel like this is generally assumed with cargo commands and doesn't need to be specified

Reply:
Removed in 74b29d2.